### ISSUE TITLE
multi-bit handling

### ISSUE THREAD

**Issue by quartzjer** [januwA/struct-buffer#3]
Given a C struct such as:
```c
typedef struct PACKED
{
  uint8_t alpha:2;
  uint8_t beta:4;
  uint8_t gamma:2;
  uint32_t data;
} example;
```
The `bits()` type doesn't appear to be able to encode multi-bit values, seems that it is intended for single-bit flags only?

**Comment by januwA** [comment]
@quartzjer  Hi, I’m not familiar with `Bit Fields`. I checked some documents and added `bitFields`, and released a version of `v5.1.0`. You can check how to [use `bitFields` here](https://github.com/januwA/struct-buffer/blob/main/test/%233.test.ts). If you have any questions, please reply to me ,Thank you for your suggestion

**Comment by quartzjer** [comment]
Amazing, perfect solution, thank you!

**Comment by quartzjer** [comment]
I think `bitFields` needs to be exported at the top level yet?

https://github.com/januwA/struct-buffer/blob/a938c088766a952914dc6dfeb9b45113291b42ec/src/index.ts#L3

**Comment by januwA** [comment]
Sorry, I forgot to export, I fixed it in `v5.1.1`
